crates/next-core/src/next_import_map.rs

```diff
@@ -66,6 +67,25 @@ pub fn get_next_client_import_map(
     import_map.cell()
 }
 
+/// Computes the Next-specific client import map.
+#[turbo_tasks::function]
+pub fn get_next_build_import_map(project_path: FileSystemPathVc) -> ImportMapVc {
+    let mut import_map = ImportMap::empty();
+
+    let package_root = attached_next_js_package_path(project_path);
+
+    insert_package_alias(
+        &mut import_map,
+        &format!("{VIRTUAL_PACKAGE_NAME}/"),
+        package_root,
+    );
+
+    import_map.insert_exact_alias("next", ImportMapping::External(None).into());
+    import_map.insert_wildcard_alias("next/", ImportMapping::External(None).into());
```

Comment:
Does this mean that `npm install next` after startup won't rebuild the config?

Reply:
Yeah. Not perfect, but it's for performance reasons. We also do that in the SSR code. Next pulls in so much stuff, which would add 2 seconds to the bundling and emitting. Mostly because it pulls in all the build related stuff too like webpack, loaders, plugins and the whole caniuse db...

We should probably optimize that in next.js to not depend on everything, but I guess it's fine for now.
When updating next.js you get a new turbopack binary anyway, which requires a restart.